### DISCUSSION TITLE
Is PHP 8 required?

### DISCUSSION THREAD

**Discussion by jor3l** [jplhomer/laravel-axiom#2] [Q&A]
Hello there, I'm running an old laravel project and the server is using PHP 7.2, was wondering if 8 is actually required, if so, I can go ahead and update the server and docker and all that, but if is not really a requirement maybe we can update the composer? I could fork and change it there but supporting that will be a lot of extra work. Thanks

**Comment by jplhomer** [comment]
Hey! I'm not sure to be honest. I don't _think_ I'm using anything PHP 8 specific - it's pretty small surface area. The bigger question would be any packages upstream e.g. the logger that requires it.
